client/src/components/visualisation/ConfigMenu.jsx

```diff
@@ -0,0 +1,253 @@
+import React, { PropTypes } from 'react';
+import DashSelect from '../common/DashSelect';
+import ColumnMenu from './configMenu/ColumnMenu';
+import LabelColumnMenu from './configMenu/LabelColumnMenu';
+import LabelInput from './configMenu/LabelInput';
+
+
+const getDatasetArray = datasetObject => {
+  const datasetArray = [];
+  const sortFunction = (a, b) => a.name.toLowerCase() > b.name.toLowerCase();
```

Comment:
Good point - the current function does sort the datasetObjects correctly via coercion, but will be inefficient with a lot of objects. Will change.